test/ci.sh

```diff
@@ -55,16 +55,13 @@ main() {
   os="$(uname -s)"
   binfile=ramalama.py
   chmod +x ${binfile} install.py
+  env
```

Comment:
What is this doing?  Is this a leftover?

Reply:
This is just testing the CI environment

Reply:
It prints all the environment variables, I want to see what they are, I want to use some